### DISCUSSION TITLE
Hello everyone

### DISCUSSION THREAD

**Discussion by ankitshah009** [OpenMined/courses#48] [Introduce Yourself]
I am Ankit Shah, a Ph.D. candidate in Language Technologies at Carnegie Mellon University. Feel free to get in touch with me - https://ankitshah009.github.io/

**Comment by yemikifouly** [comment]
Hi Ankit! 🙂 Welcome to the discussion board! Language Technologies! That sounds very interesting. How did you come to be interested in Private AI? Are you working on a project that relates to it?

**Comment by iamtrask** [comment]
Hi Ankit! Welcome!
